privatelink/azure/debug-connectivity.sh

```diff
@@ -103,7 +103,7 @@ fmt="%-5s %s\n"
 
 # shellcheck disable=SC2001
 httpsname="https://$(echo "$bootstrap" | sed -e 's/:.*//')/kafka/v3/clusters/lkc-test"
-httpsout=$(curl --silent --include "$httpsname")
+httpsout=$(curl --silent --include --connect-timeout 15 "$httpsname")
```

Comment:
I forgot to ask this, but can we use `--max-time` instead? I always prefer one big limit to many small ones: https://everything.curl.dev/usingcurl/timeouts#maximum-time-allowed-to-spend

Reply:
Done